프로그래머스/lv1/76501. 음양 더하기/음양 더하기.py

```diff
@@ -0,0 +1,5 @@
+def solution(absolutes, signs):
+    result = []
+    for n in range(len(signs)):
+        result.append(absolutes[n] * 1) if signs[n] else result.append(absolutes[n] * -1)
```

Comment:
같은 의미의 코드가 중복되어 나타납니다

Reply:
```suggestion
        result.append(absolutes[n]) if signs[n] else result.append(-absolutes[n])
```
*1 *-1 부분을 위처럼 수정하면 될까요..?

Reply:
글자 그대로 중복되는 부분이 있습니다. 그리고 그게 의미상으로도 중복입니다.

Reply:
혹시 이것도 의미상 중복이 된 코드가 있을까요?

```python
def solution(absolutes, signs):
    sum = 0
    for num, b in zip(absolutes, signs):
        if b:
            sum += num
        else:
            sum -= num
    return sum

```